hack/dind-cluster.sh

```diff
@@ -255,125 +140,247 @@ cluster's rc file to configure the bash environment:
 }
 
 function stop() {
-  echo "Cleaning up docker-in-docker containers"
+  local config_root=$1
+  local cluster_id=$2
+
+  echo "Stopping dind cluster '${cluster_id}'"
 
-  local master_cid="$(${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}")"
+  local master_cid
+  master_cid="$(${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}")"
   if [[ "${master_cid}" ]]; then
-    ${DOCKER_CMD} rm -f "${master_cid}"
+    ${DOCKER_CMD} rm -f "${master_cid}" > /dev/null
   fi
 
-  local node_cids="$(${DOCKER_CMD} ps -qa --filter "name=${NODE_PREFIX}")"
+  local node_cids
+  node_cids="$(${DOCKER_CMD} ps -qa --filter "name=${NODE_PREFIX}")"
   if [[ "${node_cids}" ]]; then
     node_cids=(${node_cids//\n/ })
     for cid in "${node_cids[@]}"; do
-      ${DOCKER_CMD} rm -f "${cid}"
+      ${DOCKER_CMD} rm -f "${cid}" > /dev/null
     done
   fi
 
-  echo "Cleanup up configuration to avoid conflict with a future cluster"
+  # Cleaning up configuration to avoid conflict with a future cluster
   # The container will have created configuration as root
-  sudo rm -rf ${CONFIG_ROOT}/openshift.local.*
+  sudo rm -rf "${config_root}"/openshift.local.*
```

Comment:
Then do

    sudo rm -rf "${config_root}"/openshift.local.config "${config_root}"/openshift.local.etcd

It's just scary to "rm -rf" a glob pattern. Especially as root.